### PR TITLE
Setup github actions to run pytest.

### PR DESCRIPTION
**Pull request by doriath**
This is copied from beangulp repository.

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      fail-fast: false
+      matrix:
+        python: ['3.6', '3.7', '3.8', '3.9']
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - run: pip install -r requirements.txt pytest
+      - run: python -m pytest beanprice
```
Comment (dnicolodi) [inline .github/workflows/test.yaml:18]: beanprice?

**Review by dnicolodi — COMMENTED** [review]
Not very important, but I prefer the first lone of commit messages not to end with a full stop, and starting the body of the commit message with 'This...' is a bit redundant as of course what is written there applied to this commit. In this case the attribution can also be simply dropped.

File: .github/workflows/test.yaml (hunk 1 above)
Comment (doriath) [reply .github/workflows/test.yaml]: Ah, sorry, I am also working on beanprice a little and I mixed them here.

**Comment by doriath** [comment]
I updated the commit message to remove the body of the comment.

**Pull request merged**